src/test/java/seedu/address/logic/parser/FilterCommandParserTest.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_PHONE_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_TAG_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.PHONE_DESC_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.PHONE_DESC_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_HIGH_RISK;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_LOW_RISK;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_MEDIUM_RISK;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FilterCommand;
+import seedu.address.model.person.PersonHasFeaturePredicate;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+
+public class FilterCommandParserTest {
+    private FilterCommandParser parser = new FilterCommandParser();
+
+    private PersonHasFeaturePredicate highTagOnlyPredicate =
+          new PersonHasFeaturePredicate(new Tag(VALID_TAG_HIGH_RISK), null);
+    private PersonHasFeaturePredicate lowTagOnlyPredicate =
+          new PersonHasFeaturePredicate(new Tag(VALID_TAG_LOW_RISK), null);
+
+    private PersonHasFeaturePredicate mediumTagOnlyPredicate =
+          new PersonHasFeaturePredicate(new Tag(VALID_TAG_MEDIUM_RISK), null);
+
+    private PersonHasFeaturePredicate phoneOnlyPredicate =
+          new PersonHasFeaturePredicate(null, new Phone(ALICE.getPhone().value));
+
+    private PersonHasFeaturePredicate phoneAndTagPredicate =
+          new PersonHasFeaturePredicate(new Tag(VALID_TAG_HIGH_RISK), new Phone(ALICE.getPhone().value));
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
```

Comment:
```suggestion
    parse_emptyArg_throwsParseException_invalidCommandFormat() {
```

Perhaps the test name could be more specific by mentioning the error message.